release-notes-4.0.0.md

```diff
@@ -4,8 +4,9 @@
 ${version-number}
 
 #### New Features
-- SCMOD-12707: Added support for pausing and resuming jobs.
+- SCMOD-12730: Added support for pausing and resuming jobs.
   - Three new endpoints have been added to pause, resume, and get the status of a job.
+  - A new `stowed_task` table has been added to the database to stow tasks belonging to a paused job.
```

Comment:
Drop this.  Too internal for release notes.  The "Added support for pausing and resuming jobs." covers it.